go/libkb/secwords.go

```diff
@@ -30,6 +35,28 @@ func secWordListN(n int) ([]string, error) {
 	return res, nil
 }
 
+func validPhrase(p string, entropy int) error {
+	numWords := secWordCount(entropy)
```

Comment:
that's a typo and you get an error

On Mon, Dec 14, 2015 at 3:54 PM, Maxwell Krohn <notifications@github.com>
wrote:

> In go/libkb/secwords.go
> <https://github.com/keybase/client/pull/1529#discussion_r47564071>:
>
> > @@ -30,6 +35,28 @@ func secWordListN(n int) ([]string, error) {
> >  	return res, nil
> >  }
> >
> > +func validPhrase(p string, entropy int) error {
> > +	numWords := secWordCount(entropy)
>
> what happens if you mistakenly type 2 spaces between a word?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1529/files#r47564071>.
>